_includes/tags_list.html

```diff
@@ -1,23 +1,21 @@
 {% assign tags = include.tags | split:'|' | sort | uniq %}
 
 {% if tags.size > 0 %}
-<footer>
   <div class="tag-list">
-    {% if tags.size == 1 %}
-      <div class="meta">Tag</div>
-    {% else %}
-      <div class="meta">Tags</div>
-    {% endif %}
+    <ul>
+      {% if tags.size == 1 %}
+        <div class="meta">Tag</div>
+      {% else %}
+        <div class="meta">Tags</div>
```

Comment:
Let's fix it even more with `li` instead of `div`:
```html
      {% if tags.size == 1 %}
        <li class="meta">Tag</li>
      {% else %}
        <li class="meta">Tags</li>
      {% endif %}
```